requirements.txt

```diff
@@ -0,0 +1,4 @@
+numpy
+openai-whisper
+torch==2.3.0
```

Comment:
Either we should add a specific version to all of them or none of them. I vote for all of them, even though they should work for now and into the future.

Reply:
Specifik till alla blir bra